TCMSApp/src/client/index.html

```diff
@@ -39,6 +39,10 @@
         </div>
     </div>
 
+
+    <script src="//code.jquery.com/jquery-1.11.3.min.js"></script>
+    <script src="https://api.trello.com/1/client.js?key=50c269d155e10e2d93a95a16c3b2d9b3"></script>
```

Comment:
and of course we need to invent the way not to store key here (ideally - so that every project creator can get own key)

but it is a new task (you can create it)